sacred/observers/file_storage.py

```diff
@@ -53,7 +55,15 @@ def __init__(
             template = basedir / "template.html"
             if not template.exists():
                 template = None
-        self.initialize(basedir, resource_dir, source_dir, template, priority)
+        self.initialize(
+            basedir,
+            resource_dir,
+            source_dir,
+            template,
+            priority,
+            no_duplicate,
```

Comment:
I would prefer something like `copy_artifacts` instead of `no_duplicate`. At least for me, it is not clear that `no_duplicate` is related to artifacts

Reply:
Yep makes sense.